src/components/bulkConcepts/component/PreviewCard.jsx

```diff
@@ -44,11 +44,12 @@ PreviewCard.propTypes = {
   closeModal: PropTypes.func.isRequired,
   addConcept: PropTypes.func.isRequired,
   concept: PropTypes.shape({
-    display_name: PropTypes.string.isRequired,
-    descriptions: PropTypes.array.isRequired,
```

Comment:
Arrays take a shape describing what it contains. Check out this [tread](https://stackoverflow.com/questions/32325912/react-proptype-array-with-shape)

Reply:
@dessHub Thanks for pointing this out. I think that I'll create a ticket for implementing props validation properly. The scope of this ticket was only to remove console warnings and errors during development mode.

Reply:
Then its fine

Reply:
Sorry for the tread